h/api/models/document.py

```diff
@@ -37,7 +37,7 @@ def title(self):
         titles = [m.value for m in self.meta if m.type == 'title']
 
         if titles:
-            return titles[0]
+            return titles[0][0]
```

Comment:
The reason why it is an array is due to the fact that there _can_ be multiple values depending on the type of value we store. The annotator [uses arrays](https://github.com/openannotation/annotator/blob/v1.2.x/src/plugin/document.coffee#L68-L81).
Then again, things are not as consistent, the title for example is [never an array](https://github.com/openannotation/annotator/blob/v1.2.x/src/plugin/document.coffee#L83-L97).